ui/app/components/app/dai-migration-component/dai-migration-notification.container.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+import DaiMigrationNotification from './dai-migration-notification.component'
+import withTokenTracker from '../../../helpers/higher-order-components/with-token-tracker'
+import { getSelectedAddress, getDaiV1Token } from '../../../selectors/selectors'
+
+const mapStateToProps = (state) => {
+  const userAddress = getSelectedAddress(state)
+  const oldDai = getDaiV1Token(state)
+
+  return {
+    userAddress,
+    token: oldDai,
+  }
+}
+
+export default compose(
+  connect(mapStateToProps),
+  withTokenTracker,
```

Comment:
I was going to ask why we're tracking this in the UI rather than using the background state, but... it appears token balances aren't in the background? 😬 . That's too bad.

Reply:
Yeah, this super annoying—I'd like to move it into the background but that's a separate effort